src/routes/Viewport/AutoForwarder/index.tsx

```diff
@@ -70,7 +70,9 @@ const AutoForwarder = ({ docked }: { docked: Boolean }) => {
           endpointId={endpointId}
         />
       ) : (
-        <p>No endpoint selected.</p>
+        <div className={`${styles['not-selected']}`}>
+          <p>No Endpoint selected</p>
+        </div>
```

Comment:
Why is this here? This seems unrelated to this PR.

Reply:
i forgot to take it out, sorry about that

Reply:
Will you be taking it out? This seems like the last issue before this PR gets merged.